server/controllers/match.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+
+// add a match and return match ID
+function addMatch() {
```

Comment:
If this is a controller, it should also handle the request and response arguments and return an HTTP response

Reply:
Provided the controller with request and response arguments.